Mention manuals in README (#186)
VladNegara/RAiSD-AI-GUI: gui/DEVELOPER.md

Reviewer: ```suggestion
The GUI code is divided into 5 main packages:
```

Reviewer: ```suggestion
- [`components`](/gui/components/): Contains all the reusable components that are used throughout the GUI.
```

Reviewer: ```suggestion
- [`execution`](/gui/execution/): Contains the code responsible for executing commands with the RAiSD-AI tool.
```

Reviewer: ```suggestion
> [!NOTE]
> The only exception is the `environment-raisd-ai-gui.yml` file, which is located in the root of the project. 
```

> - [**model**](/gui/model/): Contains the data models used in the GUI.

Reviewer: ```suggestion
- [`model`](/gui/model/): Contains the data models used in the GUI.
```

> - [**pages**](/gui/pages/): Contains the different pages of the GUI. These are the main page, history page, and settings page.

Reviewer: ```suggestion
- [`pages`](/gui/pages/): Contains the different pages of the GUI. These are the Run page, History page, and Settings page.
```

> - [**style**](/gui/style/): Contains the stylesheets and images used to style the GUI.

Reviewer: ```suggestion
- [`style`](/gui/style/): Contains the stylesheets and images used to style the GUI.
```

> - [**window**](gui/window/): Contains the set-up window, splashscreen, and the main window of the GUI.

Reviewer: ```suggestion
- [`window`](/gui/window/): Contains the set-up window, splash screen, and the main window of the GUI.
```

This package contains the stylesheets and images used to style the GUI. The stylesheets define the visual appearance of the GUI, including colors, fonts, and layout. This separation of style from functionality allows for easier maintenance and updates to the GUI's appearance.

### window

> This package contains the window parts of the GUI. This includes the set-up window which allows the user to change the default settings, and set their initial workspace folder. It also includes the splashscreen which is shown while the app is loading. Lastly, it includes the main window which sets up the different pages of the GUI and handles how these pages are connected.

Reviewer: ```suggestion
This package contains the window parts of the GUI. This includes the set-up window which allows the user to change the default settings, and set their initial workspace folder. It also includes the splash screen, which is shown while the app is loading. Lastly, it includes the main window which sets up the different pages of the GUI and handles how these pages are connected.
```
